src/plugins/lite/plugin.ts

```diff
@@ -1085,20 +1086,43 @@ LITEPlugin.prototype = {
     return this.rejectChange(node);
   },
 
+  /**
+   * Reverts a change element (ins/del) using the revert-info attribute, which contains a serialized
+   * form of the previous change elements attributes.
+   */
+  _revertFromInfo: function(changeElement: any) {
+    // find the revert info attribute which contains a serialized version of the previous change
+    const revertInfo = changeElement.getAttribute(LITEConstants.attributes.revertInfo);
+    if (!!revertInfo) {
+      let attrs;
+      try {
+        // parse the revert-info JSON into a js object
+        attrs = JSON.parse(revertInfo);
+      } catch (e) {
+        console.warn('Could not parse akorda revert information:', revertInfo);
+      }
+      if (!!attrs) {
```

Comment:
You can continue inside the try block, it is more readable and it is not necessary to create a variable with `let`.